### ISSUE TITLE
CreateBulk scan error

### ISSUE THREAD

**Issue by napei** [facebook/ent#985]
Using the same schema as I gave in #977 (as a test of performance), `CreateBulk` seems to have an issue with parsing uuids. Creating a single record works fine.

```go
	start := time.Now()
	end := start.Add(5000 * time.Hour)
	count := 5000
	banners := make([]*ent.BannerCreate, count)

	for i := 0; i < count; i++ {
		banners[i] = client.Banner.Create().SetActive(true).SetStartDate(start).SetEndDate(end).SetOrder(0).SetStatic1(false).SetStatic2(false)
	}

	newBanners, err := client.Banner.CreateBulk(banners...).Save(ctx)
	if err != nil {
		log.Fatalln("error: create banners: ", err)
	}

	translations := make([]*ent.BannerTranslationCreate, len(newBanners))

	for i, b := range newBanners {
		translations[i] = client.BannerTranslation.Create().SetActive(true).SetBanner(b)
	}

	_, err = client.BannerTranslation.CreateBulk(translations...).Save(ctx)
	if err != nil {
		log.Fatalln("error: create translations: ", err)
	}
```
```
2020/11/26 06:59:57 Tx(4ab1fc68-3ccc-416c-92c6-10c4ec9b8b28): rollbacked
2020/11/26 06:59:57 error: create banners:  insert nodes to table "banners": sql/scan: failed scanning rows: sql: Scan error on column index 0, name "id": converting driver.Value type string ("d51d970e-af53-40a3-a8d6-6794d8a17ca8") to a int64: invalid syntax
```

I'm not too sure where this error comes from as it's running inside docker and I can't figure out how to debug that, but a single `banner.Create().SetWhatever().Save(ctx)` works perfectly fine.

**Comment by a8m** [comment]
Thanks for reporting this @napei, I think I know what's the issue. 
I'll give it a look in the upcoming hours.

**Comment by a8m** [comment]
Closed with #985. Thanks for reporting this @napei!
